content/spin/api-guides-overview.md

```diff
@@ -0,0 +1,24 @@
+title = "API Guides Overview"
+template = "spin_main"
+date = "2023-03-03T03:03:03Z"
+
+---
+
+### Spin's Hosting Capabilities and Interfaces
+
+Spin is a multi-language framework for [writing server-like applications](https://developer.fermyon.com/spin/writing-apps) that compile into WebAssembly (Wasm). The Spin framework provides capabilities for your Wasm-powered application by offering a set of interfaces. For example, making sufficient HTTP functionality available to [send outbound HTTP requests](https://developer.fermyon.com/spin/http-outbound) and more.
+
+The following table shows the status of capabilities/interfaces that you can use to build and run real-world applications with Wasm.
+
+| Host Capabilities/Interfaces           | Stability  |  Cloud  |
+|----------------------------------------|----------|-------|
+| HTTP                          | Stable   | Yes   |
+| Redis                         | Stable   | Yes   |
+| Outbound HTTP                          | Stable   | Yes   |
+| Outbound Redis                         | Stable  | Yes   |
+| Dynamic config                         | Experimental | Limited |
```

Comment:
For those unaware, what's considered "dynamic config", and why is support for it "limited" in the Cloud?

Reply:
There is a dynamic config link which I can add https://developer.fermyon.com/spin/dynamic-configuration

Reply:
Thanks @bacongobbler 
These points have been updated in the new push (latest commit)